test.sh

```diff
@@ -1 +1 @@
-java -cp build/user-agent-detector.jar net.pieroxy.ua.tooling.UserAgentTester test test-data/database.gz
+java -cp target/user_agent_detector-2015-10-31-SNAPSHOT.jar net.pieroxy.ua.tooling.UserAgentTester test src/test/resources/database.gz
```

Comment:
I just copied your version string. Actually using test.sh is not good, you might want to replace the .sh script by src/java/test/sometestclass.java and then maven will run tests on deploy/compile.